src/app/dataset-view/additional-components/metadata-component/components/form-components/event-time-field/event-time-field.component.ts

```diff
@@ -0,0 +1,69 @@
+import { ChangeDetectionStrategy, Component, OnInit } from "@angular/core";
+import { BaseField } from "../base-field";
+import { EventTimeSourceKind } from "../../add-polling-source/add-polling-source-form.types";
+import { FormBuilder, FormGroup } from "@angular/forms";
+import { RxwebValidators } from "@rxweb/reactive-form-validators";
+
+@Component({
+    selector: "app-event-time-field",
+    templateUrl: "./event-time-field.component.html",
+    styleUrls: ["./event-time-field.component.sass"],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class EventTimeFieldComponent extends BaseField implements OnInit {
```

Comment:
The name of the component is inadequate - we have a selection of date formats, not showing "event time" field.

Reply:
Renamed component on SelectDateFormatFieldComponent